app/helpers/parse_dd20.py

```diff
@@ -829,6 +830,8 @@ def parse_dd20_excelsheets_to_dataframe(
     header_index: int = 1,
     sheetname_linedata: str = "Linjedata - Sommer",
     sheetname_stationsdata: str = "Stationsdata",
+    line_data_valid_format_hash_value: str = "eb087129d0c2413ee9768957c2c847ae", 
+    station_data_valid_format_hash_value: str = "91006da47706518d76724105a97dfb61" 
```

Comment:
```suggestion
    line_data_valid_hash: str = "eb087129d0c2413ee9768957c2c847ae", 
    station_data_valid_hash: str = "91006da47706518d76724105a97dfb61" 
```
Just an opinion, but i think these variable names are equally specific ("value" is generic - everything is a value?) and format does not much add much in my opinion. Go with your preferred style.

Reply:
valid suggestion. I Take it :-)